src/Type/Php/InArrayFunctionTypeSpecifyingExtension.php

```diff
@@ -75,8 +80,8 @@ public function specifyTypes(FunctionReflection $functionReflection, FuncCall $n
 			|| (
```

Comment:
I haven't been able to figure out whether `$context->true()` can actually do anything. I tried commenting it out and it didn't break any tests. As far as I was able to tell, the specified types are mostly intersected with the original type, in which case widening the specified type doesn't do anything.